doc/api/buffer.md

```diff
@@ -1996,7 +1998,7 @@ console.log(copy);
 added: v0.1.90
 -->
 
-* `encoding` {string} The character encoding to decode to. **Default:** `'utf8'`.
+* `encoding` {string} The character encoding to use. **Default:** `'utf8'`.
```

Comment:
It might only be me, but the former description is somewhat more specific for me and therefore easier to understand.